R/pkg/DESCRIPTION

```diff
@@ -35,3 +35,4 @@ Collate:
     'sparkR.R'
     'stats.R'
     'utils.R'
+RoxygenNote: 5.0.0
```

Comment:
I'm not sure we need to modify this file in this PR - It encodes some dependencies among R files which is sometimes fragile.